cmd/ebpf/xdp/n3n6_entrypoint.c

```diff
@@ -85,44 +86,45 @@ static __always_inline __u8 match_sdf_filter_ipv4(struct packet_context *ctx, st
         return 0;
     }
 
-
+    upf_printk("SDF Filter values:");
```

Comment:
Also forgot to mention in the commit message that I added logs for SDF filter values